docs/section22/Rule22-8.md

```diff
@@ -0,0 +1,54 @@
+
+# CHW&CW - Rule 22-8  
+
+**Rule ID:** 22-8  
+**Rule Description:** For Baseline chilled water system with cooling capacity of 300 tons or more, the secondary pump shall be modeled with variable-speed drives.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.10 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13.
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+3. Pass Rule-22-7.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13: `PLACEHOLDER`
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water: `if B_RMR.ASHRAE229.chillers:`
+3. Pass Rule-22-7: `if rule-22-7 == "PASS":`
+
+## Rule Logic:  
+
+- For each pump in B_RMR: `for pump_b in B_RMR.ASHRAE229.pumps:`
+
+  - Save loop served by pump and pump to a dictionary: `loop_pump_dictionary[pump_b.loop_or_piping].append(pump)`
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.ASHRAE229.chillers:`
+
+  - Get cooling loop that chiller servers: `chw_loop_b = chiller_b.cooling_loop`
+
+    - Add chiller rated capacity to CHW loop total capacity: `chw_loop_capacity_dict[chw_loop_b] += chiller_b.rated_capacity`
+
+- For each CHW loop that chiller serves: `for loop_b in chw_loop_capacity_dict.keys():`
+
+  - Check if CHW loop total cooling capacity is 300 tons or more: `if chw_loop_capacity_dict[loop_b] >= 300 * CONVERSION(TON_TO_BTUH):`
+
+    - For each secondary loops served by CHW loop: `for secondary_loop_b in loop_b.child_loops:`
+
+      - For each secondary pump associated with secondary loop: `for secondary_pump in loop_pump_dictionary[secondary_loop_b]:`
+
```

Comment:
@JasonGlazer I have created an issue for this in the RMR repo as well- we have two data elements in pump data group 'is_variable_speed' and 'PumpSppednControlOptions' for 'fixed_speed' and 'variable_speed'. we should remove one of them.

Reply:
Agreed. I also noted it under one of the HW RDSs.